plugin/trino-prometheus/src/main/java/io/trino/plugin/prometheus/PrometheusClient.java

```diff
@@ -52,7 +53,7 @@
 {
     static final Type TIMESTAMP_COLUMN_TYPE = createTimestampWithTimeZoneType(3);
     static final String METRICS_ENDPOINT = "/api/v1/label/__name__/values";
-    private static final OkHttpClient httpClient = new Builder().build();
+    private static OkHttpClient httpClient;
```

Comment:
You don't need to remove `final`.

Reply:
I think we need to, since we can only instantiate the client inside the constructor. 
If we leave `final` in this variable declaration, the assignment at https://github.com/anishasthana/trino/commit/2bb294c9a04237340770bd562ee21128e631f10e#diff-4d0317b4037cbc623979423299663ef08163e175b38b0a8226376d61c7b07adeR69 will fail

Reply:
Please remove `static` instead.